src/inflation/model/price.clj

```diff
@@ -0,0 +1,8 @@
+(ns inflation.model.price)
+
+(defn new-price [country type value year month]
```

Comment:
Avoid functions that have several positional parameters, they are hard to extend and make the client code difficult to read. In this case the client code can just create the map (and you don't need this function).